modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java

```diff
@@ -695,14 +697,7 @@ else if (type.equals(DDMFormFieldTypeConstants.RICH_TEXT)) {
 
 					richTextValues.add(richTextValue);
 
-					if (richTextValue.length() >
-							_SORTABLE_TEXT_FIELDS_TRUNCATED_LENGTH) {
-
-						richTextValue = richTextValue.substring(
-							0, _SORTABLE_TEXT_FIELDS_TRUNCATED_LENGTH);
-					}
-
-					truncatedValues.add(richTextValue);
+					truncatedValues.add(_truncate(richTextValue));
```

Comment:
@adolfopa I think we could rename it to `textValue` to make it more clear.

Reply:
I've separated both cases in the conditional, so I'm keeping the names as is.

Reply:
I've separated these two cases in the conditional, so I believe the rename is no longer necessary.